### PR TITLE
Cu 866aezbc4

### PR DESCRIPTION
**Pull request by vaionexbusra**

### CODE REVIEW

File: src/components/Layout/footer/footer.js [hunk 1]
```diff
@@ -1,7 +1,35 @@
...
+      console.log(response)
+      if (response.ok) {
+        console.log('Subscription successful!')
+        setEmail('')
+      } else {
+        console.log('Subscription failed. Please try again.')
+      }
+    } catch (error) {
+      console.log(error)
+      console.log('An error occurred. Please try again.')
```
Comment (vaionexbusra) [inline src/components/Layout/footer/footer.js:30]: Should I add toastify as it is in Legaliser?

File: src/components/Layout/footer/footer.js [hunk 2]
```diff
@@ -99,23 +127,32 @@ export function Footer() {
...
-              type="email"
-              name="email-address"
-              id="email-address"
-              autoComplete="email"
-              required
-              className="w-full min-w-0 appearance-none rounded-md border-0 bg-white/5 px-3 py-1.5 text-base text-white shadow-sm ring-1 ring-inset ring-white/10 placeholder:text-gray-500 focus:ring-2 focus:ring-inset focus:ring-indigo-500 sm:w-56 sm:text-sm sm:leading-6"
-              placeholder="Enter your email"
-            />
-            <div className="mt-4 sm:mt-0 sm:ml-4 sm:flex-shrink-0">
+            <div className=" min-w-0 inline-flex justify-start ring-1 ring-inset ring-white/10 border-0 bg-white/5 rounded-md focus-within:ring-2 focus-within:ring-indigo-500">
```
Comment (vaionexbusra) [inline src/components/Layout/footer/footer.js:137]: subscribe button and input are in the same line? Will it be changed it? or keep it as the original version?

File: src/components/Layout/footer/footer.js (hunk 1 above)
Comment (paulsp94) [reply src/components/Layout/footer/footer.js:30]: yes please

File: src/components/Layout/footer/footer.js (hunk 2 above)
Comment (paulsp94) [reply src/components/Layout/footer/footer.js:137]: make it as we discussed it today for legaliser

**Review by paulsp94 — APPROVED** [review]
LGTM

**Pull request merged**